apps/fulfillment-e2e/src/support/page_fragments/picking-in-progress-modal.fragment.ts

```diff
@@ -1,6 +1,7 @@
 export class PickingInProgressModalFragment {
   getWrapper = () => cy.get('oryx-picking-in-progress-modal');
 
+  getProceedToPickingBtn = () => cy.contains('Proceed to picking');
```

Comment:
Should we get the proper button, instead of via text? This could change in the future.